tiledb/api/c_api/attribute/attribute_api.cc

```diff
@@ -135,7 +135,6 @@ int32_t tiledb_attribute_get_cell_size(
 int32_t tiledb_attribute_dump(
     const tiledb_attribute_handle_t* attr, FILE* out) {
   ensure_attribute_is_valid(attr);
-  ensure_cstream_handle_is_valid(out);
```

Comment:
Why remove this?

Reply:
This validation needs to stay.

And resolving conversations that raise concerns silently, without fixing nor even commenting, is poor collaboration practice.